Sources/SwiftSyntax/Convenience.swift

```diff
@@ -45,3 +45,57 @@ extension MemberAccessExprSyntax {
     )
   }
 }
+
+extension EnumCaseParameterSyntax {
```

Comment:
I _think_ that writing a script that does this automatically for an arbitrary file would be a nice exercise and an example of how to use SwiftSyntax!

How would you feel about having an additional script that `./format.py` would invoke? It could live as an executable target in `swift-format` as well, perhaps?

Or.. maybe that could be a part of `swift-format` that would only trigger on files with paths matching a setting? I.e. 

```
{
  // ...
  "rules" : {
    "AllPublicDeclarationsHaveDocumentation" : false,
    "AlwaysUseLowerCamelCase" : true,
    
    // What if a rule would support not only a Boolean value, but an array of paths for which the value is true (or different from a default value)
    "AlphabetizeDeclarations": [ "Sources/SwiftSyntax/generated/Convenience.swift", "**/Convenience.swift"]
    // ...
  },
  // ... 
}
```

I could start with a little experiment as an example to learn to use SwiftSyntax, and if I can get a prototype, I could write a proposal on Swift Forums to make the settings support path arrays?

Reply:
We discussed a similar idea [here](https://github.com/apple/swift-syntax/pull/1903#discussion_r1262293841). At the time, we concluded that it wouldn't be a trivial task, so the subject was put on the back burner. However, the fact that the idea has resurfaced suggests there might be a real need for such a tool, and it could be beneficial to revisit the discussion.

How about we move this discussion to a separate issue? @natikgadzhi, would you like to do the honors and open one? ☺️

Reply:
If you have an idea of how to implement it, I’d be happy to take it. But as I mentioned in the other discussion, I think it’s far from a trivial task, if not even (nearly) impossible. 

And so far, we have been good with these comments so at least as far as I’m concerned there’s no immediate need for action.